js/renovation/ui/editors/__tests__/check_box.test.tsx

```diff
@@ -199,6 +199,47 @@ describe('CheckBox', () => {
     describe('Effects', () => {
       afterEach(clearEventHandlers);
 
+      describe('updateIconFontSize', () => {
+        it('should set icon font size on init', () => {
+          const checkBox = new CheckBox({ iconHeight: 22, iconWidth: 22 });
+          checkBox.iconRef = { current: { style: {} } } as any;
+          const icon = checkBox.iconRef.current;
+
+          checkBox.updateIconFontSize();
+
+          expect(icon?.style.fontSize).toEqual('16px');
+        });
+
+        it('should change icon font size after runtime changes in "iconWidth"/"iconHeight" options', () => {
+          const checkBox = new CheckBox({ iconHeight: 22, iconWidth: 22 });
+          checkBox.iconRef = { current: { style: {} } } as any;
+          const icon = checkBox.iconRef.current;
+
+          checkBox.updateIconFontSize();
+
+          checkBox.props.iconHeight = 44;
+          checkBox.props.iconWidth = 66;
+
+          checkBox.updateIconFontSize();
+
+          expect(icon?.style.fontSize).toEqual('32px');
+        });
+
+        it('should correctly change icon font size if "iconWidth"/"iconHeight" options are not numbers', () => {
+          const checkBox = new CheckBox({ iconHeight: '22px', iconWidth: '22px' });
+          checkBox.iconRef = React.createRef() as any;
+          checkBox.iconRef.current = {
+            offsetHeight: 22,
+            offsetWidth: 22,
+            style: {},
+          } as any;
+          checkBox.updateIconFontSize();
+
+          const icon = checkBox.iconRef.current;
+          expect(icon?.style.fontSize).toEqual('16px');
+        });
+      });
+
```

Comment:
what about values differ to number and pixels?

Reply:
just realized this test is always green. actually idk how to check other values here. but there are some qunits tests with em and % valus